lib/core.js

```diff
@@ -534,24 +534,34 @@ impress.gsInterval = () => {
 };
 
 // Unload configuration and stop server
-impress.stop = () => {
+impress.stop = callback => {
   impress.cache.clear();
   const servers = impress.servers;
   if (!servers) {
     impress.log.warn('No servers active');
+    callback();
     return;
   }
+
+  let count = 0;
```

Comment:
Can we maybe use `metasync.each()` here instead of manual counting?

Reply:
`metasync.each()` iterates `Array` but we need to iterate `Object`, we can take keys but I think counter is ok here

Reply:
As another option, we can use `Object.values()` here with `metasync.each()`.

Reply:
Ok